queue/queue.py

```diff
@@ -1,15 +1,96 @@
 class Queue:
-  def __init__(self):
-    self.size = 0
-    # what data structure should we
-    # use to store queue elements?
-    self.storage = 
-
-  def enqueue(self, item):
-    pass
-  
-  def dequeue(self):
-    pass
-
-  def len(self):
-    pass
+    def __init__(self):
+        self.size = 0
+        # what data structure should we
+        # use to store queue elements?
+        self.storage = LinkedList()
+
+    def enqueue(self, item):
+        self.storage.add_to_tail(item)
```

Comment:
yes! this is exactly what we were going for :) great job making that connection!